soc/arm/st_stm32/stm32f0/Kconfig.defconfig.stm32f030x4

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2019 Antony Pavlov <antonynpavlov@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32F030X4
+
+# N.B. stm32cube/stm32f0xx/soc/stm32f0xx.h know nothing
```

Comment:
This likely requires an update of stm32cube package.
I will try to provide that next week.

Reply:
@erwango 

> This likely requires an update of stm32cube package.
> I will try to provide that next week.

I have examined stm32cube (https://www.st.com/en/embedded-software/stm32cubef0.html). It looks like zephyr already uses latest stm32cubef0 version.